Makefile.am

```diff
@@ -5732,13 +5732,27 @@ dist_noinst_DATA += \
     m4 \
     contrib/sssd.spec.in \
     contrib/sssd.sysusers \
```

Comment:
Shouldn't this line be removed?

Reply:
I don't know. I didn't touch it because we don;t use those files on Gentoo.

Reply:
Ok, yeah, you're right, so I fixed that.